nbs/core.ipynb

```diff
@@ -67,7 +67,7 @@
     "from neuralforecast.models import (\n",
     "    GRU, LSTM, RNN, TCN, DilatedRNN,\n",
```

Comment:
Would we want to add a unit test of `fit`, `predict` new random seed argument.
To guarantee replicability of their expected results in the future?